server/interceptor/interceptor.go

```diff
@@ -274,13 +277,20 @@ func isLeafOK(leaf *trillian.QueuedLogLeaf) bool {
 	return leaf == nil || leaf.Status == nil || leaf.Status.Code == int32(codes.OK)
 }
 
-// serviceName returns "some.package.service" for
-// "/some.package.service/method".
+var fullyQualifiedRE = regexp.MustCompile(`^/([\w\.]+)/(\w+)$`)
```

Comment:
idle query: does the dot need backslash-escaping inside a character class?

Reply:
looks like it doesn't.  Thanks for catching that.